src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -240,6 +240,14 @@ Docker builds Telescope's dependencies at launch and keeps them on disk. In some
 
 :::
 
+### Seeding PostgresSQL
+
+The first time you start the database, it doesn't have any data. Therefore, we need database seeding as a way to populate the database with an initial set of data to make some parts of our application work properly.
+
+To seed the database with initial data use the command `pnpm seed`.
```

Comment:
I think what you need to run is `pnpm db:init`, which runs `db:migrate` and `db:seed`. It'd be a good idea to explain what those 2 commands do, and also it's very likely that, after running `pnpm db:init` the parse container needs to be restarted.

Reply:
Working on it.